apps/application-system/form/src/auth.ts

```diff
@@ -19,7 +22,7 @@ if (userMocked) {
     redirectPathSilent: '/silent/signin-oidc',
     authority: environment.identityServer.authority,
     client_id: 'island-is-1',
-    scope: `openid profile api_resource.scope ${ApplicationScope.read} ${ApplicationScope.write} ${UserProfileScope.read} ${NationalRegistryScope.individuals}`,
+    scope: `openid profile api_resource.scope ${ApplicationScope.read} ${ApplicationScope.write} ${UserProfileScope.read} ${NationalRegistryScope.individuals} ${TemporaryVoterRegistryScope.read} ${PartyLetterRegistryScope.read} ${EndorsementScope.listWrite} ${EndorsementScope.listRead} ${EndorsementScope.read} ${EndorsementScope.write}`,
```

Comment:
I'm wondering if converting this for an array would be more readable and less repetitive of string interpolation and using a join to make the string within the auth library